nilearn/maskers/surface_labels_masker.py

```diff
@@ -95,7 +95,7 @@ class SurfaceLabelsMasker(TransformerMixin, CacheMixin, BaseEstimator):
 
     def __init__(
         self,
-        labels_img,
+        labels_img=None,
```

Comment:
So this seems to pass the `check_parameters_default_constructible` test, but I am not quite sure what complies the best with sklearn guidelines i.e.

whether to have `labels_img` as a parameter to `.fit()` or maintain our old behavior of letting users pass it during instantiation (but now it won't throw any error if they don't)

Reply:
Guidelines here: https://scikit-learn.org/stable/developers/develop.html#instantiation

Reply:
If we do this we must thrown an error at fit if labels_img is None.

I think it'd be good to do this for all maskers: looking at NiftiMasker (which I believe is the oldest and most canonical) it allows None for the maask image.

Reply:
It's because the mask image can be computed from the data when using `NiftiMasker`.
To me it's OK to provide `labels_img` at instantiation time rather than fit time.